playbooks/libvirt/openshift-cluster/tasks/launch_instances.yml

```diff
@@ -23,6 +23,12 @@
     creates: '{{ os_libvirt_storage_pool_path }}/{{ image_name }}'
   when: image_compression in ["xz"] and downloaded_image.changed
 
+- name: Uncompress TGZ Base Cloud image
+  command: 'tar zxvf {{ os_libvirt_storage_pool_path }}/{{ [image_name, image_compression] | join(".") }}'
+  args:
+    creates: '{{ os_libvirt_storage_pool_path }}/{{ image_name }}'
+  when: image_compression in ["tgz"] and downloaded_image.changed
```

Comment:
I guess we should also handle gz format. For example, centos atomic is provided as .qcow.gz
I am adding it.